appmap/test/data/django/app/settings.py

```diff
@@ -10,5 +10,3 @@
 # Must set ROOT_URLCONF else we get
 # AttributeError: 'Settings' object has no attribute 'ROOT_URLCONF'
 ROOT_URLCONF = "app.urls"
-
-MIDDLEWARE = ["appmap.django.Middleware"]
```

Comment:
Why was this removed?  Is it because it's expected that appmap will change this variable to insert `"appmap.django.Middleware"` in it ?

Reply:
Yes, exactly. By not adding the middleware manually, the tests that use this app confirm that it's automatically injected.